### PR TITLE
Fix Exchanging and Receiving translation in swap page. fix #390

### LINKED ISSUE #390: Exchanging and Receiving words translated on Brazilian Portuguese file but not on Rwallet
photo_2020-08-07_18-17-50

### PR DESCRIPTION
**Pull request by imstar15**
image

### CODE REVIEW

File: src/common/translations/pt-BR.json [hunk 1]
```diff
@@ -136,7 +136,9 @@
         "have": "Eu tenho %{balance} na carteira %{wallet}",
         "min": "MIN",
         "half": "METADE",
-        "all": "TUDO"
+        "all": "TUDO",
+        "exchanging": "Troca",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:140]:
```suggestion
        "exchanging": "Trocando",
```

**Comment by bitcoinuser** [comment]
Hi @chrisli30, don't you think make more sense use the word Sending instead of Exchanging? At least on Portuguese, exchanging looks strange. Thanks.

**Comment by chrisli30** [comment]
> Hi @chrisli30, don't you think make more sense use the word Sending instead of Exchanging? At least on Portuguese, exchanging looks strange. Thanks.

@bitcoinuser We will keep the english Exchanging, if you think it's not common in Portuguese, we show Sending in Portuguese.

Could you update the Exchanging value to "Sending" in Portuguese?

File: src/common/translations/pt.json [hunk 2]
```diff
@@ -136,7 +136,9 @@
         "have": "Eu tenho %{balance} no %{wallet}",
         "min": "MIN",
         "half": "HALF",
-        "all": "TUDO"
+        "all": "TUDO",
+        "exchanging": "Troca",
```
Comment (bitcoinuser) [inline src/common/translations/pt.json:140]:
```suggestion
        "exchanging": "Enviando",
```

File: src/common/translations/pt-BR.json [hunk 3]
```diff
@@ -136,7 +136,9 @@
         "have": "Eu tenho %{balance} na carteira %{wallet}",
         "min": "MIN",
         "half": "METADE",
-        "all": "TUDO"
+        "all": "TUDO",
+        "exchanging": "Trocando",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:140]:
```suggestion
        "exchanging": "Enviando",
```

**Comment by bitcoinuser** [comment]
> > Hi @chrisli30, don't you think make more sense use the word Sending instead of Exchanging? At least on Portuguese, exchanging looks strange. Thanks.
> 
> @bitcoinuser We will keep the english Exchanging, if you think it's not common in Portuguese, we show Sending in Portuguese.
> 
> Could you update the Exchanging value to "Sending" in Portuguese?

Done.

**Pull request merged**